src/MCPServer/lib/server/shared_dirs.py

```diff
@@ -0,0 +1,329 @@
+    </preconfiguredChoice>
+    <!-- Document empty directories -->
+    <preconfiguredChoice>
+      <appliesTo>d0dfa5fc-e3c2-4638-9eda-f96eea1070e0</appliesTo>
+      <goToChain>65273f18-5b4e-4944-af4f-09be175a88e8</goToChain>
+    </preconfiguredChoice>
+    <!-- Send transfer to quarantine -->
+    <preconfiguredChoice>
+      <appliesTo>755b4177-c587-41a7-8c52-015277568302</appliesTo>
+      <goToChain>d4404ab1-dc7f-4e9e-b1f8-aa861e766b8e</goToChain>
+    </preconfiguredChoice>
+  </preconfiguredChoices>
+</processingMCP>
+"""
+
+AUTOMATED_PROCESSING_CONFIG = """<processingMCP>
+  <preconfiguredChoices>
+    <!-- Store DIP -->
+    <preconfiguredChoice>
+      <appliesTo>5e58066d-e113-4383-b20b-f301ed4d751c</appliesTo>
+      <goToChain>8d29eb3d-a8a8-4347-806e-3d8227ed44a1</goToChain>
+    </preconfiguredChoice>
+    <!-- Select compression level -->
+    <preconfiguredChoice>
+      <appliesTo>01c651cb-c174-4ba4-b985-1d87a44d6754</appliesTo>
+      <goToChain>414da421-b83f-4648-895f-a34840e3c3f5</goToChain>
+    </preconfiguredChoice>
+    <!-- Examine contents -->
+    <preconfiguredChoice>
+      <appliesTo>accea2bf-ba74-4a3a-bb97-614775c74459</appliesTo>
+      <goToChain>e0a39199-c62a-4a2f-98de-e9d1116460a8</goToChain>
+    </preconfiguredChoice>
+    <!-- Perform file format identification (Submission documentation & metadata) -->
+    <preconfiguredChoice>
+      <appliesTo>087d27be-c719-47d8-9bbb-9a7d8b609c44</appliesTo>
+      <goToChain>4dec164b-79b0-4459-8505-8095af9655b5</goToChain>
+    </preconfiguredChoice>
+    <!-- Normalize (match 1 for "Normalize for preservation") -->
+    <preconfiguredChoice>
+      <appliesTo>cb8e5706-e73f-472f-ad9b-d1236af8095f</appliesTo>
+      <goToChain>612e3609-ce9a-4df6-a9a3-63d634d2d934</goToChain>
+    </preconfiguredChoice>
+    <!-- Normalize (match 2 for "Normalize for preservation") -->
+    <preconfiguredChoice>
+      <appliesTo>7509e7dc-1e1b-4dce-8d21-e130515fce73</appliesTo>
+      <goToChain>612e3609-ce9a-4df6-a9a3-63d634d2d934</goToChain>
+    </preconfiguredChoice>
+    <!-- Bind PIDs -->
+    <preconfiguredChoice>
+      <appliesTo>05357876-a095-4c11-86b5-a7fff01af668</appliesTo>
+      <goToChain>fcfea449-158c-452c-a8ad-4ae009c4eaba</goToChain>
+    </preconfiguredChoice>
+    <!-- Create SIP(s) -->
+    <preconfiguredChoice>
+      <appliesTo>bb194013-597c-4e4a-8493-b36d190f8717</appliesTo>
+      <goToChain>61cfa825-120e-4b17-83e6-51a42b67d969</goToChain>
+    </preconfiguredChoice>
+    <!-- Delete packages after extraction -->
+    <preconfiguredChoice>
+      <appliesTo>f19926dd-8fb5-4c79-8ade-c83f61f55b40</appliesTo>
+      <goToChain>85b1e45d-8f98-4cae-8336-72f40e12cbef</goToChain>
+    </preconfiguredChoice>
+    <!-- Transcribe files (OCR) -->
+    <preconfiguredChoice>
+      <appliesTo>7079be6d-3a25-41e6-a481-cee5f352fe6e</appliesTo>
+      <goToChain>1170e555-cd4e-4b2f-a3d6-bfb09e8fcc53</goToChain>
+    </preconfiguredChoice>
```

Comment:
This should be:

```xml
    <!-- Transcribe files (OCR) -->
    <preconfiguredChoice>
      <appliesTo>82ee9ad2-2c74-4c7c-853e-e4eaf68fc8b6</appliesTo>
      <goToChain>0a24787c-00e3-4710-b324-90e792bfb484</goToChain>
    </preconfiguredChoice>
```

Reply:
Doh, I guess the auto merge when rebasing missed a few things. Changed now.